### DISCUSSION TITLE
Executable vs Portable vs Windows App?

### DISCUSSION THREAD

**Discussion by e-t-l** [adirh3/Fluent-Search#355] [Q&A] [answered]
Are there any pros or cons I'm missing regarding the different methods of installing this program? Is it known that some version, e.g. Windows Store App, is more or less resource-efficient, or gets updated sooner?

**Comment by adirh3** [comment, marked as answer]
> Are there any pros or cons I'm missing regarding the different methods of installing this program? Is it known that some version, e.g. Windows Store App, is more or less resource-efficient, or gets updated sooner?

There are 4 distributions - Exe, Appx, Portable, Windows Store (also Appx)

Exe - most recommended, has all features, includes all release feeds (Stable/Nightly), auto-updates.
Portable - same as EXE, it may have some issues with auto updating.
Appx - Changing theme based on Windows does not work, includes all release feeds, not recommended since I plan to deprecate it.
Windows Store - same as Appx, is updated with Stable release feed, does not support Nightly, auto updates through Microsoft Store.
